wgpu-types/src/lib.rs

```diff
@@ -110,7 +128,33 @@ bitflags::bitflags! {
         /// but it is not yet implemented.
         ///
         /// https://github.com/gpuweb/gpuweb/issues/696
-        const ANISOTROPIC_FILTERING = 0x01;
+        const ANISOTROPIC_FILTERING = 0x0000_0000_0001_0000;
+        /// Extensions which are part of the upstream webgpu standard
+        const WEBGPU_EXTENSIONS = 0x0000_0000_0000_FFFF;
```

Comment:
I wonder if we can rename this to avoid repeating ourself (in "extensions"). How do you feel about `ALL_WGPU`, `ALL_UNSAFE`, and `ALL_NATIVE`?

Reply:
Good by me.